### DISCUSSION TITLE
Does the menu url can be opened as a new web page?

### DISCUSSION THREAD

**Discussion by Triple-Z** [adityatelange/hugo-PaperMod#824] [Q&A] [answered]
https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#add-menu-to-site

Is there something configuration to make `target="_blank"` possible?

**Comment by adityatelange** [comment, marked as answer]
No configuration is present, however you can customize the below portion of code to achieve that 

- https://github.com/adityatelange/hugo-PaperMod/blob/15b984bd215b7c4a73385e76486ec93b0aadcc53/layouts/partials/header.html#L108-L117

- [Override theme template](https://github.com/adityatelange/hugo-PaperMod/wiki/FAQs#override-theme-template)

**Comment by slashtechno** [comment]
I got external links to open in a new tab by replacing lines 129 and 130 of `layouts/partials/header.html`
https://github.com/adityatelange/hugo-PaperMod/blob/eba92edd83acb8a6a679cd58d440e22d5c31f822/layouts/partials/header.html#L129-L130
with 
```html
 <a href="{{ .URL | absLangURL }}" {{ if strings.HasPrefix .URL "http" }} target="_blank"{{ end }} title="{{ .Title | default .Name }} {{- cond $is_search (" (Alt + /)" | safeHTMLAttr) ("" | safeHTMLAttr ) }}"
 {{- cond $is_search (" accesskey=/" | safeHTMLAttr) ("" | safeHTMLAttr ) }}>
```
(I created a custom layout)
